frameworks/react/package.json

```diff
@@ -25,7 +25,7 @@
   "devDependencies": {
     "@babel/preset-env": "^7.12.10",
     "@babel/preset-typescript": "^7.12.7",
-    "@cloudinary/base": "^1.0.0-beta.2",
+    "@cloudinary/base": "^1.0.0-beta.3",
```

Comment:
@newmaniese Can you please remove the version bumps? we have an automation that takes care of this:)